client/src/components/tools/item/crafts.jsx

```diff
@@ -28,7 +28,9 @@ const Recipe = ({ recipe, index }) => {
           {Object.values(recipe.ingredients).map((ingredient, i) => (
             <List.Item key={`ing_${index}_${i}`}>
               <List.Content>
-                <Image src={images.item(ingredient.id)} />
+                <a href={`/tools/item/${ingredient.id}`}>
```

Comment:
I believe you want to use `Link` from `@reach/router` instead of a regular `a` for internal links.

Reply:
Added in 1cf6b17